aws-logs-resourcepolicy/aws-logs-resourcepolicy.json

```diff
@@ -0,0 +1,63 @@
+{
+  "typeName": "AWS::Logs::ResourcePolicy",
+  "description": "The resource schema for AWSLogs ResourcePolicy",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs.git",
+  "properties": {
+    "PolicyName": {
+      "description": "A name for resource policy",
+      "type": "string",
+      "pattern": "^([^:*\\/]+\\/?)*[^:*\\/]+$",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "PolicyDocument": {
+      "description": "The policy document",
+      "type": "string",
+      "pattern": "[\\u0009\\u000A\\u000D\\u0020-\\u00FF]+",
+      "minLength": 1,
+      "maxLength": 5120
+    }
+  },
+  "additionalProperties": false,
+  "primaryIdentifier": [
+    "/properties/PolicyName"
+  ],
+  "createOnlyProperties": [
+    "/properties/PolicyName"
+  ],
+  "handlers": {
+    "create": {
+      "permissions": [
+        "logs:PutResourcePolicy",
+        "logs:DescribeResourcePolicies"
+      ]
+    },
+    "read": {
+      "permissions": [
+        "logs:DescribeResourcePolicies"
+      ]
+    },
+    "update": {
+      "permissions": [
+        "logs:PutResourcePolicy",
+        "logs:DescribeResourcePolicies",
+        "logs:DeleteResourcePolicy"
+      ]
+    },
+    "delete": {
+      "permissions": [
+        "logs:DescribeResourcePolicies",
```

Comment:
I don’t think delete needs DescribeResourcePolicies unless something in the backend requires it. The deleteHandler itself doesn’t do any checking if a resourcepolicy exists or not